docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.8"
+services:
+  server:
+    build: .
+    image: ottomated/crewlink-server:build
+    container_name: crewlinkserver
+    ports:
+        - 9736:9736
+        # - 443:443 # Uncomment this if you enable HTTPS
+    expose:
+        - 9736
+        # - 443 # Uncomment this if you enable HTTPS
+    environment: 
+        # ADDRESS: Specify the server domain
+        # NAME: Specify the server name
+        # HTTPS: Enables https. You must place privkey.pem and fullchain.pem in your CWD.
+        # SSLPATH: Specifies an alternate path to SSL certificates.
```

Comment:
This might be picky, but it might be a good idea to format this as a list instead of a map, to keep it consistent with the other ports/expose elements.

Reply:
I don't fully agree with that (I usually prefer structured data to strings that need to be parsed) but here `key: value` would already be transformed into `key=value` so I understand what you mean. Fixed :)